pkgs/desktops/deepin/go-package/dde-daemon/default.nix

```diff
@@ -0,0 +1,229 @@
+    ];
+    "accounts/manager.go" = [
+      # /usr/share/dde-daemon/accounts/dbus-udcp.json
+    ];
+    "image_effect/utils.go" = [
+      [ "runuser" "${util-linux.bin}/bin/runuser" ]
+    ];
+    "misc/usr/share/deepin/scheduler/config.json" = [
+      [ "/usr/bin" "/run/current-system/sw/bin" ]
+      # path to deepin-anything/dde-file-manager
+    ];
+  };
+in
+buildGoPackage rec {
+  pname = "dde-daemon";
+  version = "5.14.122";
+
+  goPackagePath = "github.com/linuxdeepin/dde-daemon";
+
+  src = fetchFromGitHub {
+    owner = "linuxdeepin";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-KoYMv4z4IGBH0O422PuFHrIgDBEkU08Vepax+00nrGE=";
+  };
+
+  patches = [
+    ./0001-fix-wrapped-name-for-verifyExe.patch
+    ./0002-fix-PATH-when-was-launched-by-dbus.patch
+  ];
+
+  postPatch = replaceAll "/usr/lib/deepin-api" "/run/current-system/sw/lib/deepin-api"
+    + replaceAll "/usr/lib/deepin-daemon" "/run/current-system/sw/lib/deepin-daemon"
+    + replaceAll "/usr/share/wallpapers" "/run/current-system/sw/share/wallpapers"
+    + replaceAll "/usr/share/backgrounds" "/run/current-system/sw/share/backgrounds"
+    + replaceAll "/bin/bash" "${runtimeShell}"
+    + replaceAll "/bin/sh" "${runtimeShell}"
+    + replaceAll "/usr/bin/setxkbmap" "${xorg.setxkbmap}/bin/setxkbmap"
+    + replaceAll "/usr/bin/xdotool" "${xdotool}/bin/xdotool"
+    + replaceAll "/usr/bin/getconf" "${getconf}/bin/getconf"
+    + replaceAll "/usr/bin/dbus-send" "${dbus}/bin/dbus-send"
+    + replaceAll "/usr/share/zoneinfo" "${tzdata}/share/zoneinfo"
+    + replaceAll "/usr/share/X11/xkb/rules/base.xml" "${xkeyboard_config}/share/X11/xkb/rules/base.xml"
+    + replaceAll "/usr/bin/kwin_no_scale" "kwin_no_scale"
+    + replaceAll "/usr/bin/deepin-system-monitor" "deepin-system-monitor"
+    + replaceAll "/usr/bin/dde-launcher" "dde-launcher"
+    + replaceAll "/usr/bin/deepin-calculator" "deepin-calculator"
+    + replaceAll "/usr/bin/systemd-detect-virt" "systemd-detect-virt"
+    + getUsrPatchFrom patchList + ''
+    patchShebangs .
+  '';
+
+  goDeps = ./deps.nix;
+
+  nativeBuildInputs = [
+    pkg-config
+    deepin-gettext-tools
+    gettext
+    python3
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    go-dbus-factory
+    go-gir-generator
+    go-lib
+    dde-api
+    linux-pam
+    libxcrypt
+    alsa-lib
+    glib
+    libgudev
+    gtk3
+    gdk-pixbuf-xlib
+    networkmanager
+    libinput
+    libnl
+    librsvg
+    pulseaudio
+    tzdata
+    xkeyboard_config
+    util-linux
+    ddcutil
+  ];
+
+  buildPhase = ''
+    runHook preBuild
+    GOPATH="$GOPATH:${go-dbus-factory}/share/gocode"
+    GOPATH="$GOPATH:${go-gir-generator}/share/gocode"
+    GOPATH="$GOPATH:${go-lib}/share/gocode"
+    GOPATH="$GOPATH:${dde-api}/share/gocode"
+    make -C go/src/${goPackagePath}
+    runHook postBuild
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    make install DESTDIR="$out" PREFIX="/" -C go/src/${goPackagePath}
```

Comment:
Same here. (Or makeFlags, it applies to both the buildPhase and the installPhase)